bookshop/src/store/index.js

```diff
@@ -39,70 +43,81 @@ const store = new Vuex.Store({
   actions: {
     async booklist ({commit}) {
       try {
-        let {data: {data: res}} = await axios.get('http://localhost:8008/api/getBooks')
-        commit('set', {type: 'books', items: res})
+        let {data: {data: res}} = await axios.get(config.api_url + 'getBooks')
+        commit('set', {type: 'books', data: res})
+      } catch (e) {
+
+      }
+    },
+    async getBook ({commit}, {id}) {
+      try {
+        let {data: {data: res}} = await axios.get(config.api_url + 'getBookById/' + id)
+        commit('set', {type: 'book', data: res})
       } catch (e) {
 
       }
     },
     async genres ({commit}) {
       try {
-        let {data: {data: res}} = await axios.get('http://localhost:8008/api/getGenres')
-        commit('set', {type: 'genres', items: res})
+        let {data: {data: res}} = await axios.get(config.api_url + 'getGenres')
+        commit('set', {type: 'genres', data: res})
       } catch (e) {
 
       }
     },
     async authors ({commit}) {
       try {
-        let {data: {data: res}} = await axios.get('http://localhost:8008/api/getAuthors')
-        commit('set', {type: 'authors', items: res})
+        let {data: {data: res}} = await axios.get(config.api_url + 'getAuthors')
+        commit('set', {type: 'authors', data: res})
       } catch (e) {
 
       }
     },
-    async login ({commit, dispatch}, creds) {
+    async login ({commit, dispatch}, {payload}) {
       let token = null
-      if (_.isObject(creds)) {
+      if (_.isString(payload.login) && _.isString(payload.password)) {
         try {
-          let response = await axios.post('http://localhost:8008/api/auth', creds)
+          let response = await axios.post(config.api_url + 'auth', {login: payload.login, password: payload.password})
           token = _.get(response, 'data.data.token')
           if (token) {
-            dispatch('getuser', token)
+            dispatch('getuser', {token: token})
           }
         } catch (e) {
           token = null
         }
         if (token) {
           sessionStorage.setItem('token', token)
-          commit('set', {type: 'isLoggedIn', items: true})
+          commit('set', {type: 'isLoggedIn', data: true})
```

Comment:
here and all other files:
data -> items
use more specific naming

Reply:
???????

Reply:
ok, persuaded :)

Reply:
Ok, I`m gonna to look at this minor issue in the future.